pkgs/applications/science/biology/manta/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, cmake, zlib, python2 }:
+
+stdenv.mkDerivation rec {
+  pname = "manta";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = "manta";
+    rev = "v${version}";
+    sha256 = "1711xkcw8rpw9xv3bbm7v1aryjz4r341rkq5255192dg38sgq7w2";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ zlib python2 ];
+  postFixup = ''
+    pushd $out/lib/python
+    sed -i 's|/usr/bin/env python2|${python2}/bin/python|' makeRunScript.py
+    rm makeRunScript.pyc
+    echo "import makeRunScript" | python2
+    popd
+    pushd $out/lib/python/pyflow
+    sed -i 's|#!/usr/bin/env python|#!${python2}/bin/python|' pyflow.py
+    sed -i 's|/bin/bash|${stdenv.shell}|' pyflowTaskWrapper.py
+    rm pyflowTaskWrapper.pyc pyflow.pyc
+    echo "import pyflowTaskWrapper" | python2
+    echo "import pyflow" | python2
+    popd
```

Comment:
I think what you were intending to do was:

```
  postPatch = ''
    sed -i 's|/bin/bash|${stdenv.shell}|' lib/python/pyflow/pyflowTaskWrapper.py
  '';
  installCheck = true;
  installCheckPhase = ''
    rm $out/lib/python/**/*.pyc
    PYTHONPATH=$out/lib/python:$PYTHONPATH python -c 'import makeRunScript'
    PYTHONPATH=$out/lib/python/pyflow:$PYTHONPATH python -c 'import pyflowTaskWrapper; import pyflow'
  '';
```

Reply:
That's much cleaner, thanks! I couldn't get installCheckPhase to execute though even with installCheck set to true, so i left the patching in postFixup. Is this ok?

Reply:
`doInstallCheck = true;` is the correct attribute. I was tired :)

Reply:
Thanks :)